src/helix-ui/elements/HXAlertElement.html

```diff
@@ -1,4 +1,4 @@
-<div id="wrapper">
+<div id="hxWrapper">
   <hx-icon id="icon" type="info-circle"></hx-icon>
   <span id="content">
```

Comment:
Every `id` and `class` used in the shadow dom will require prefixing, not just the root element.